package/submariner.sh

```diff
@@ -7,7 +7,8 @@ export CHARON_PID_FILE=/var/run/charon.pid
 rm -f ${CHARON_PID_FILE}
 
 if [ "${SUBMARINER_DEBUG}" == "true" ]; then
-    DEBUG="--debug -v=9"
+    DEBUG="-v=9"
+    export CE_IPSEC_DEBUG=true
```

Comment:
We control that separately from operator/helm scripts.

You could want to debug submariner code itself, without debugging the ipsec part (which can be very verbose), I'd keep them separate.